metricbeat/module/mysql/mysql.go

```diff
@@ -36,7 +56,7 @@ func GetMySQLEnvDSN() string {
 	dsn := os.Getenv("MYSQL_DSN")
 
 	if len(dsn) == 0 {
-		dsn = "root@tcp(127.0.0.1:3306)/"
+		dsn = CreateDSN("tcp(127.0.0.1:3306)/", "root", "")
```

Comment:
These are only used for testing. The reason these functions are in this file is because they are needed here is because the test files are always only compiled from one package, means the helpers in the module_test are not available in metricset_test. Perhaps these should be move to a file called testing.go?